hist/hist/src/THnBase.cxx

```diff
@@ -1312,6 +1332,22 @@ void THnBase::ResetBase(Option_t * /*option = ""*/)
    }
 }
 
+////////////////////////////////////////////////////////////////////////////////
+/// \brief Compute integral (sum of counts) of histogram in all dimensions
+/// \param respectAxisRange if false, count all bins including under/overflows,
+///                         if true, restrict sum to the user-set axis range
+/// \sa Projection(0)::Integral() as alternative
+Double_t THnBase::Integral(const Bool_t respectAxisRange) const
```

Comment:
There is also `THnBase::ComputeIntegral`, IIUC the main difference is that it normalizes the integral to 1 whereas this function does not. Can we maybe also document it here?